app/scripts/controllers/CreateRoomCtrl.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+    function CreateRoomCtrl (Room, $uibModalInstance) {
+        
+        
+        this.addChat = function() {  //Not sure what the correct syntax to add new chat is
+            Room.addChatRoom({text: newChatRoomName});
+         };  
+        
+        this.ok = function() {
```

Comment:
You could pass the room name to the service in this method. Simply give this function an argument and pass it's value to the $add call. 

``` javascript
this.ok = function(roomName){
  Room.addChatRoom(roomName);
 //code
}
```